taskvine/src/manager/vine_task.c

```diff
@@ -845,6 +845,16 @@ char * vine_task_to_json(struct vine_task *t)
 		}
 		buffer = string_combine(buffer, "\n");
 	}
+
+	if(t->env_list){
```

Comment:
@BarrySlyDelgado I suggest using `dttools/src/buffer.h` here, as it is more efficient and the intent is clearer.